core/store/orm/orm.go

```diff
@@ -370,8 +370,7 @@ func (orm *ORM) CreateExternalInitiator(externalInitiator *models.ExternalInitia
 func (orm *ORM) DeleteExternalInitiator(accessKey string) error {
 	orm.MustEnsureAdvisoryLock()
 	return orm.db.
-		Where("access_key = ?", accessKey).
```

Comment:
Why didn't this work?

Reply:
It works

Reply:
This change was accidentally included.  It was my first mistaken attempt at fixing the error.  The root of the problem was because we cannot have UNIQUE table constraints with soft deletes, as the constraint on the DB works for soft deletes and non-deleted items.
I solved it by removing the 'DeletedAt' field.